packages/node_modules/@webex/plugin-memberships/test/integration/spec/memberships.js

```diff
@@ -692,3 +763,26 @@ function validateMembershipEvent(event, membership, actor, creator = actor) {
   }
   debug(`membership:${event.event} event validated`);
 }
+
+/**
+ * Validate a rooms:updated event for moderated space change
+ *
+ * We check this in membership tests (instead of the rooms tests)
+ * because a change in membership is  the activity that triggers this event.
+ * @param {Object} event - rooms event
+ * @param {Object} room -- return from the API that created the room
+ * @param {Boolean} isLocked - expected value for isLocked
+ * @returns {void}
+ */
+function validateRoomsUpdatedEvent(event, room, isLocked) {
+  const {data} = event;
+
+  assert((room.id === data.id),
+    'rooms:updated event did not have the expected roomId');
+  assert((room.type === data.type),
+    'rooms:updated event did not have the expected room type');
+  assert((data.isLocked === isLocked),
+    'rooms:updated did not have expected isLocked value');
```

Comment:
Any reason this is formatted like? I think it should be fine on the same line or each param is new lined. This formatting is just weird.

Reply:
The linter complains about lines too long.

Reply:
Eh good enough for me, #FISI